packages/eslint-plugin-fiori-tools/eslintrc-typescript.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
+    overrides: [
+        {
+            'plugins': ['@typescript-eslint'],
+            'files': ['*.ts', '**/*.ts'],
+            'excludedFiles': ['*.d.ts', '**/*.d.ts'],
+            'parser': '@typescript-eslint/parser',
+            'extends': ['plugin:@typescript-eslint/recommended'],
+            'parserOptions': {
+                'project': ['./tsconfig.json']
+            },
+            'rules': {
+                '@typescript-eslint/no-unused-vars': 'off'
```

Comment:
Why do we turn it off by default?

Reply:
Because it conflicts with `eslint-plugin-fiori-custom` when it is enabled on *.ts 
https://github.com/SAP/open-ux-tools/pull/984/files#diff-87c1761fdfc506f6b4a8460a4c880050b50c02ced1f94121c594009b6837de22R7
and causes the lint command to crash parsing the component.ts on the rule `@typescript-eslint/no-unused-vars`